chia/server/ws_connection.py

```diff
@@ -335,33 +335,34 @@ async def send_messages(self, messages: List[Message]):
         for message in messages:
             await self.outgoing_queue.put(message)
 
+    async def _wait_and_retry(self, msg: Message, queue: asyncio.Queue):
+        try:
+            await asyncio.sleep(1)
+            await queue.put(msg)
+        except Exception as e:
+            self.log.debug(f"Exception {e} while waiting to retry sending rate limited message")
+            return
+
     async def _send_message(self, message: Message):
         encoded: bytes = bytes(message)
         size = len(encoded)
         assert len(encoded) < (2 ** (LENGTH_BYTES * 8))
         if not self.outbound_rate_limiter.process_msg_and_check(message):
             if not is_localhost(self.peer_host):
-                self.log.debug(
-                    f"Rate limiting ourselves. message type: {ProtocolMessageTypes(message.type).name}, "
-                    f"peer: {self.peer_host}"
-                )
-
+                reason = "peer is localhost"
```

Comment:
looks like reason is only used in the else case

Reply:
Thanks, Almog